bin/build.sh

```diff
@@ -277,6 +286,27 @@ if [ ${INSTALL_DEPENDENCIES} -eq 1 ] ; then
     sudo apt-get install -y ${DEPENDENCIES}
     ;;
 
+
+  linux---x86_64)
+    sudo yum -y install ninja-build
+    sudo rpm -U https://packages.microsoft.com/config/centos/7/packages-microsoft-prod.rpm
+    sudo yum -y install dotnet-sdk-5.0
+    sudo pip3 install pyyaml psutil toml
```

Comment:
We can do `yum install python3-yaml` stuff like we do `apt-get install python3-yaml` on Debian OSes.

Reply:
I did not manage to do this on the Linux version I was testing this on. In particular, python3-yaml package is not there.